pydanticrud/backends/dynamodb.py

```diff
@@ -303,9 +303,15 @@ def query(self, query_expr: Optional[Rule] = None, filter_expr: Optional[Rule] =
                                            "expression")
 
         else:
-            resp = table.scan(**params)
-
-        return [self._deserialize_record(rec) for rec in resp["Items"]]
+            try:
+                resp = table.scan(**params)
+            except ClientError as e:
+                if e.response["Error"]["Code"] == "ResourceNotFoundException":
+                    resp = None
```

Comment:
You should probably make a similar exception handler after line 302 in case the call to `query` also raises a ClientError

Reply:
done